examples/main_variables/type_string.c

```diff
@@ -5,7 +5,7 @@ int main(void)
 {
     adjust_init();
     ADJUST_CONST_STRING(a, "\"adjust.h\" can ignore annoying '\"' \t marks!\n");
-    ADJUST_VAR_STRING(b, "To infinity... and beyond!");
+    ADJUST_VAR_STRING(b, "Tok infinity... and beyond!");
```

Comment:
I'm fine with modifications made to the test files, but I'd rather not include typos.